app/actions/week_to_goal_action.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class WeekToGoalAction
+  def calculate(objective)
+    return 999 if objective.account.automatic_topup_configs.blank?
```

Comment:
If it's blank, this number does not make sense. But 999 does make sense. Can we return something that does not make sense instead of 999?